### PR TITLE
#263 language field

### PR DESCRIPTION
**Pull request by naveen2507**
- Issue came up while testing with description field (fixed)

### CODE REVIEW

**Comment by naveen2507** [comment]
@reckart can you review this please

File: inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplIntegrationTest.java [hunk 1]
```diff
@@ -813,7 +813,7 @@ public void listProperties_WithASavedConceptAndNotAll_ShouldFindOneConcept() {
             .hasSize(1)
             .element(0)
             .hasFieldOrPropertyWithValue("identifier", handle.getIdentifier())
-            .hasFieldOrPropertyWithValue("name", handle.getName())
+            .hasFieldOrProperty("name")
```
Comment (reckart) [inline inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplIntegrationTest.java]: Why not include the value anymore in the test?

File: inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplIntegrationTest.java [hunk 2]
```diff
@@ -1240,8 +1240,8 @@ public void getConceptRoots_WithWildlifeOntology_ShouldReturnRootConcepts() thro
                 .map(KBHandle::getName);
 
         String[] expectedLabels = {
-            "Adaptation", "Animal Intelligence", "Collection", "Conservation Status", "Ecozone",
-            "Habitat", "Red List Status", "Taxon Name", "Taxonomic Rank"
+            "Adaptation", "AnimalIntelligence", "Collection", "ConservationStatus", "Ecozone",
```
Comment (reckart) [inline inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplIntegrationTest.java]: Why were the spaces removed from the expected labels?

File: inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplLanguageIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,602 @@
...
+import de.tudarmstadt.ukp.inception.kb.graph.KBProperty;
+import de.tudarmstadt.ukp.inception.kb.model.KnowledgeBase;
+import de.tudarmstadt.ukp.inception.kb.reification.Reification;
+import de.tudarmstadt.ukp.inception.kb.util.TestFixtures;
+
+@RunWith(Parameterized.class)
+@SpringBootTest(classes = SpringConfig.class)
+@Transactional
+@DataJpaTest
+public class KnowledgeBaseServiceImplLanguageIntegrationTest  {
```
Comment (reckart) [inline inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplLanguageIntegrationTest.java]: Why is there a new test class for the language stuff? It seems to duplicate the existing integration test.

File: inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/util/TestFixtures.java [hunk 4]
```diff
@@ -65,7 +65,6 @@ public KnowledgeBase buildKnowledgeBase(Project project, String name, Reificatio
         kb.setLabelIri(RDFS.LABEL);
         kb.setPropertyTypeIri(RDF.PROPERTY);
         kb.setDescriptionIri(RDFS.COMMENT);
-        kb.setDefaultLanguage("en");
```
Comment (reckart) [inline inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/util/TestFixtures.java]: I believe that I had added the language there. Why did you remove it?

Comment (naveen2507) [reply inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/util/TestFixtures.java]: @reckart I have created another Junit test with language enabled.

File: inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplLanguageIntegrationTest.java (hunk 3 above)
Comment (naveen2507) [reply inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplLanguageIntegrationTest.java]: Without language it takes the name from the IRI , but with language it supports the label. I did that to have a test for language .

File: inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplIntegrationTest.java (hunk 2 above)
Comment (naveen2507) [reply inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplIntegrationTest.java]: Without language it takes the name from the IRI , but with language it supports the label.

File: inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplLanguageIntegrationTest.java (hunk 3 above)
Comment (naveen2507) [reply inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplLanguageIntegrationTest.java]: I will add tests for some specific use cases in #440 . It doesnt duplicate everything, as I have removed most of the methods, except the one which checks the name info.

Comment (Rentier) [reply inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplLanguageIntegrationTest.java]: Please do not create a new unit test for every aspect to test, but add new tests to the existing one. I would just add 'withLanguage' to the test name that use language or group them together with Java 5 magic. https://junit.org/junit5/docs/current/user-guide/#writing-tests-nested

Comment (naveen2507) [reply inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplLanguageIntegrationTest.java]: @Rentier okay . I will keep the changes for #440 .
